.github/workflows/builder.yaml

```diff
@@ -0,0 +1,49 @@
+---
+name: "build-docker-image"
+
+on:
+  pull_request:
+
+  push:
+    tags:
+      - "v*"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        platform: [ linux/amd64, linux/arm64 ]
+
+    steps:
+      - name: "Checkout the repository"
+        uses: actions/checkout@v4
+
+      - name: "Extract metadata for the Docker image"
+        if: github.event_name == 'push' && github.ref_type == 'tag' # Only extract metadata if we push a tag
+        id: metadata_extraction
+        uses: docker/metadata-action@v5
+        with:
+          images: ghcr.io/${{ github.repository }}
+
+      - name: "Login to GitHub Container Registry"
+        if: github.event_name == 'push' && github.ref_type == 'tag' # Only login if we push a tag
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: "Set up Docker Buildx"
```

Comment:
"Buildx"?

Reply:
Buildx is their new build environment (if that's what it's called). It allows us to create multi-architecture images.